source-map-uploader-plugin.js

```diff
@@ -23,6 +23,8 @@ class BugsnagSourceMapUploaderPlugin {
     this.overwrite = options.overwrite
     this.endpoint = options.endpoint
     this.ignoredBundleExtensions = options.ignoredBundleExtensions || ['.css']
+    this.minifiedUrl = options.minifiedUrl
+    this.minifyBundleUrls = options.minifyBundleUrls
```

Comment:
Do we need to do this ourselves? The regex looks a bit risky that it might pickup other files, not just the hash, and it feels like it might not work for everyone.

Our CLI supports `bundle` and `bundleUrl` already – can we not just expose these to be set in config and keep it to users to pass in wildcards according to their own requirements?